config/install/message_ui.settings.yml

```diff
@@ -0,0 +1,6 @@
+update_tokens:
+  update_tokens: false
+  how_to_act: 'update_when_added'
+  number_items: 250
+show_preview: true
+anonymous: 'Anonymous' # What is this variable and how best to store/access it?
```

Comment:
That is a comment I added at the start, I'll remove the comment. We use this setting at: `/src/Form/MessageForm.php:57`